peercrawler_html_server.py

```diff
@@ -27,9 +27,18 @@ def bg_thread_func():
     peerman.crawl(forever=True, delay=60)
 
 
+def refresh_node_info():
+    global nodes
+
+    try:
+        nodes = get("https://nano.community/data/representative-mappings.json").json()
```

Comment:
Shouldn't the variable `nodes` be protected with a mutex?

Reply:
Since there's only a single thread modifying it at a time, I don't think that's needed.

Reply:
It is very likely safe as it is but let's put a mutex anyway just so that we do not even have to ask the question.
https://stackoverflow.com/questions/2291069/is-python-variable-assignment-atomic